attributes/mod_auth_openid.rb

```diff
@@ -0,0 +1,17 @@
+# mod_auth_openids
+default['apache']['mod_auth_openid']['ref']        = 'v0.8'
```

Comment:
we're adding new attributes?

Reply:
Nope, this was move from default attributes. 

Mostly so I can remember to either remove it or add it to another cookbook later on.